Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Base image
+FROM php:7.3-fpm as base
+
+WORKDIR /usr/src/app
+
+RUN pecl install apcu && \
```

Comment:
Codacy found an issue: [Delete the apt-get lists after installing something](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6062620)